src/openrct2/config/Config.h

```diff
@@ -26,7 +26,7 @@ struct GeneralConfiguration
 {
     // Paths
     utf8* rct1_path;
-    utf8* rct2_path;
+    std::string rct2_path{};
```

Comment:
`{}` is not necessary for `std::string`.

```suggestion
    std::string rct2_path;
```